core/api/transport/impl/ws/ws_session.cpp

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "api/transport/impl/ws/ws_session.hpp"
+
+#include <cstring>
+
+#include <boost/asio/dispatch.hpp>
+#include <boost/config.hpp>
+#include <outcome/outcome.hpp>
```

Comment:
```suggestion
#include "outcome/outcome.hpp"
```
I know it's strange but outcome/outcome.hpp is actually in core/.